README.md

```diff
@@ -261,12 +261,21 @@ highlighted like regular clojure code.
 ### Highlight markdown syntax in docstrings
 
 By default markdown syntax is highlighted in the docstrings using
-`markdown_inline` grammar. To disable this feature set
+`markdown-inline` grammar. To disable this feature set
 
 ``` emacs-lisp
 (setopt clojure-ts-use-markdown-inline nil)
 ```
 
+### Highlight regex syntax
```

Comment:
I'd probably add one of your images here, so people would know what font-locking comes from this feature. (and probably we should add a markdown screenshot in the section above down the road)

Reply:
I added screenshots for both, markdown and regex syntax.